eo/Dockerfile

```diff
@@ -15,5 +15,7 @@ RUN umask 0000 && conda env update -f /environment.yml
 # (https://github.com/conda-forge/gdal-feedstock/issues/83#issue-162911573)
 ENV CPL_ZIP_ENCODING=UTF-8
 
+COPY notebook_config.yml.template /notebook_config.yml.template
```

Comment:
Ok like I said, I can now really see that we use the template as is, so it's not a template. One cannot expect some variables to be replaced like in the birdhouse-deploy config folder.